index.html

```diff
@@ -209,17 +206,17 @@ <h6 class="languages-world">Languages</h6>
 
                     <!-- FRAMEWORKS AND SKILLS -->
                     <div class="frame-Skill">
-                    <div class="frame-skl-container">
-                        <p class="frOrSkill">Frameworks</p>
-                        <nav><i class="fa fa-angle-right" aria-hidden="true"></i></nav>
-                    </div>
+                    <ul class="frame-skl-container">
+                        <li class="frOrSkill">Frameworks</li>
+                        <li><i class="fa fa-angle-right" aria-hidden="true"></i></li>
+                    </ul>
                     <hr>
 
                     <!-- SKILLS -->
-                    <div class="frame-skl-container">
-                        <p class="frOrSkill">Skills</p>
-                        <nav><i class="fa fa-angle-right" aria-hidden="true"></i></nav>
-                    </div>
+                    <ul class="frame-skl-container">
+                        <li class="frOrSkill">Skills</li>
+                        <li><i class="fa fa-angle-right" aria-hidden="true"></i></li>
+                    </ul>
                     <hr>
```

Comment:
- [ ] Good job so far 👍 but all the technology categories (languages, frameworks, and skills) should be in a single `ul` tag and each of them in their `li`. Kindly correct this.
- [ ] (OPTIONAL) Kindly arrange the font size of languages to match that of frameworks and skills and also align their `arrows` just like in the template. Yours are too close.
- [ ] (OPTIONAL) Also, kindly apply the correct font color for a better user interface 😄 
- Your design:
![Annotation 2022-03-24 200205](https://user-images.githubusercontent.com/85113567/159992854-3f6aa003-3dea-4c6e-9297-ed3915341a37.jpg)
- Figma design:
![Annotation 2022-03-24 200455](https://user-images.githubusercontent.com/85113567/159992891-3a467e74-7f85-463a-9fce-877e6d99d6dd.jpg)